WordPress/WordPressTest/BlockEditorSettingsServiceTests.swift

```diff
@@ -85,8 +84,9 @@ class BlockEditorSettingsServiceTests: XCTestCase {
     }
 
     private func validateThemeResponse() {
+        let siteID = blog.dotComID ?? 0
```

Comment:
I assume it's okay to fallback to `0` for the id if none is found, otherwise you wouldn't have gone through the trouble of doing it. 

I'm curious about why that's the case 🙂

Reply:
These test doesn't really care what the siteID is we just care that the correct path is used. That path includes the siteID. Originally (before using the BlogBuilder I hardcoded it to 1).